website_crm_quick_answer/README.rst

```diff
@@ -34,7 +34,7 @@ To use this module, you need to:
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/186/10.0
+   :target: https://runbot.odoo-community.org/runbot/186/12.0
```

Comment:
Please generate the README instead

Reply:
@cristinamartinrod please review this comment